### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by dumitrumunteanu**

### CODE REVIEW

File: app/Models/Post.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 
-class Post extends Model
+class Post extends Model implements LoggableInterface
```
Comment (egorbwork) [inline app/Models/Post.php:8]: It is name of your Article model?

File: app/Services/DebugRequestActivityLog.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Http\Request;
+
+class DebugRequestActivityLog extends AbstractRequestActivityLogger {
+    public function debugRequestData(Request $request): array {
+        return [
+            'ip' => $request->ip(),
```
Comment (egorbwork) [inline app/Services/DebugRequestActivityLog.php:13]: mistake

File: app/Services/ProductionRequestActivityLog.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Http\Request;
+
+class ProductionRequestActivityLog extends AbstractRequestActivityLogger {
+    public function debugRequestData(Request $request): array {
+        return $request->all();
```
Comment (egorbwork) [inline app/Services/ProductionRequestActivityLog.php]: mistake

**Review by egorbwork — CHANGES_REQUESTED** [review]
The ModelLogger part looks correct. But creation of RequestLogger is invalid for the moment. Content looks reversed and does not look as one is extension of another one.

File: app/Models/Post.php (hunk 1 above)
Comment (dumitrumunteanu) [reply app/Models/Post.php:8]: Yes, it is.

**Review by egorbwork — APPROVED** [review]
Now looks fine.

**Pull request closed without merging**